temp_name_lib/src/star_physics/local_values/gravity.rs

```diff
@@ -0,0 +1,40 @@
+use super::local_value;
```

Comment:
The setup `star_physics/local_values/*.rs' seems overly complicated. All of these functions could be in one file only.